jr-stree/src/main/java/com/fasterxml/jackson/jr/stree/JacksonJrsTreeCodec.java

```diff
@@ -28,10 +28,11 @@ public JacksonJrsTreeCodec(ObjectCodec codec) {
     }
 
     // @since 2.17
-    public void setFailOnDuplicateKeys(boolean state) {
-        _failOnDuplicateKeys = state;
+    public void setFailOnDuplicateKeys(boolean failOnDuplicateKeys,boolean useBigDouble) {
```

Comment:
you cant change a public API in a patch? revert this. You can add extra methods but you cant change the signature of a public method without deprecating it at least a minor release.

Reply:
Removed!